READMEs/consume-flow.md

```diff
@@ -0,0 +1,172 @@
+<!--
+Copyright 2022 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Marketplace Flow
+
+This quickstart describes a batteries-included flow including using off-chain services for metadata (Aquarius) and consuming datasets (Provider).
+
+It focuses on Alice's experience as a publisher, and Bob's experience as a consumer.
+
+Here are the steps:
+
+1.  Setup
+2.  Alice publishes data asset
+3.  Bob downloads it
+
+Let's go through each step.
+
+## 1. Setup
+
+### First steps
+
+To get started with this guide, please refer to [datatokens-flow](datatokens-flow.md) and complete the following steps :
+- [x] Setup : Prerequisites
+- [x] Setup : Download barge and run services
+- [x] Setup : Install the library from v4 sources
+- [x] Setup : Set envvars
+
+## 2. Publish Data NFT & Datatoken
+
+In your project folder (i.e. my_project from `Install the library` step) and in the work console where you set envvars, run the following:
+
+Please refer to [datatokens-flow](datatokens-flow.md) and complete the following steps :
+- [x] 2.1 Create an ERC721 data NFT
+
+Then in the same python console:
+```python
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+
+# Specify metadata and services, using the Branin test dataset
+date_created = "2021-12-28T10:55:11Z"
+
+metadata = {
+    "created": date_created,
+    "updated": date_created,
+    "description": "Branin dataset",
+    "name": "Branin dataset",
+    "type": "dataset",
+    "author": "Trent",
+    "license": "CC0: PublicDomain",
+}
+
+# ocean.py offers multiple file types, but a simple url file should be enough for this example
+from ocean_lib.structures.file_objects import UrlFile
+url_file = UrlFile(
+    url="https://raw.githubusercontent.com/trentmc/branin/main/branin.arff"
+)
+
+# Encrypt file(s) using provider
+encrypted_files = ocean.assets.encrypt_files([url_file])
+
+
+# Publish asset with services on-chain.
+# The download (access service) is automatically created, but you can explore other options as well
+asset = ocean.assets.create(
+    metadata,
+    alice_wallet,
+    encrypted_files,
+    erc20_templates=[1],
+    erc20_names=["Datatoken 1"],
+    erc20_symbols=["DT1"],
+    erc20_minters=[alice_wallet.address],
+    erc20_fee_managers=[alice_wallet.address],
+    erc20_publish_market_order_fee_addresses=[ZERO_ADDRESS],
+    erc20_publish_market_order_fee_tokens=[ocean.OCEAN_address],
+    erc20_caps=[ocean.to_wei(100000)],
+    erc20_publish_market_order_fee_amounts=[0],
+    erc20_bytess=[[b""]],
+)
+
+did = asset.did  # did contains the datatoken address
+print(f"did = '{did}'")
+
+```
+
+## 3. Bob buys data asset, and downloads it
+Now, you're Bob the data consumer.
+In usual cases, Bob buys the dataset but here, let's have Alice send hhim some tokens directly.
```

Comment:
Bob doesn't _buy_ the asset in this README. Please update these lines accordingly.